yellowbrick/features/manifold.py

```diff
@@ -203,24 +209,36 @@ def __init__(
         ax=None,
         manifold="mds",
         n_neighbors=None,
+        features=None,
+        classes=None,
         colors=None,
-        target=AUTO,
-        alpha=0.7,
+        colormap=None,
+        target_type="auto",
+        projection=2,
+        alpha=0.75,
```

Comment:
Out of curiosity, why the change from 0.7 to 0.75, to match the default in the super class?

Reply:
Yes! This was done to match the super class.